include/ignition/gui/qml/GzPose.qml

```diff
@@ -143,11 +153,37 @@ Item {
       width: parent.width
       columns: 4
 
-      Text {
-        text: 'X (m)'
-        leftPadding: 5
-        color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
-        font.pointSize: 12
+      Rectangle {
+        color: "transparent"
+        height: 40
+        Layout.preferredWidth: xText.width + 40
+        Loader {
+          id: xPlot
+          sourceComponent: gzplot
+          width: iconWidth
+          height: iconHeight
+          y:10
+        }
+        Component.onCompleted: {
+          xPlot.item.gzIconWidth = iconWidth
+          xPlot.item.gzIconHeight = iconHeight
+          xPlot.item.gzIconY = 10
+          xPlot.item.gzComponentInfo = "x"
+          xPlot.item.gzMimeData = { "text/plain" : (model === null) ? "" :
+              "Component," + model.entity + "," + model.typeId + "," +
+              model.dataType + "," + xPlot.item.gzComponentInfo + "," + model.shortName
+          }
```

Comment:
To me, this is very `gz-sim` specific (eg., `model.entity`, `model.typeId`, etc). How about we expose a variable for the mime data in `GzPose.qml` then pass the information from `gz-sim`?

What are your thoughts @chapulina ?